dds_cli/utils.py

```diff
@@ -198,7 +198,7 @@ def perform_request(
             raise dds_cli.exceptions.DDSCLIException(message=message)
 
         if response.status_code == http.HTTPStatus.FORBIDDEN:
-            if DDSEndpoint.CREATE_PROJ in endpoint:
+            if DDSEndpoint.CREATE_PROJ or DDSEndpoint.ADD_NEW_MOTD in endpoint:
```

Comment:
Need to have either `if DDSEndpoint.CREATE_PROJ in or DDSEndpoint.ADD_NEW_MOTD in endpoint:` or this:
```suggestion
            if any(ep in endpoint for ep in [DDSEndpoint.CREATE_PROJ, DDSEndpoint.ADD_NEW_MOTD]):
```
Otherwise it will just check if `DDSEndpoint.CREATE_PROJ` is not None, so it will always be true